### ISSUE TITLE
Defining OED fields for aggregate data

### ISSUE THREAD

**Issue by MattDonovan82** [OasisLMF/OpenDataStandards#84]
Linked to the following issue:
(https://github.com/OasisLMF/OasisLMF/issues/956)

**Comment by johcarter** [comment]
Currently in OED we have NumberOfBuildings and IsAggregate (true/false) for representing aggregate data, however they are not used other than for some models.  We are reviewing the requirements for disaggregating exposures to see how Oasis would use these fields, and whether we need additions.  A list of use cases is currently being prepared to investigate this.

**Comment by johcarter** [comment]
This is how we suggest use of the fields NumberOfBuildings and IsAggregate might be used in combination to describe different types of location level data. 
![image](https://user-images.githubusercontent.com/11421733/189919356-508ee21a-0c29-468e-8df7-4ceff9423e52.png)

We don't think we would see a case of aggregate data combined with multiple buildings per site which would require an extra field to capture both number of risks (for aggregate data) and number of buildings (per site) within a single record in the location file. Therefore we don't think we need to change or add any more fields to handle aggregate data.

Comments welcome.

**Comment by aiste-kalinauskaite** [comment]
That's fine. I assume that if IsAggregate field isn't present, then it will default to cases 1, 3 and 5, depending on what NumberOfBuildings field is populated as?

**Comment by johcarter** [comment]
Yes @aiste-kalinauskaite thats correct. And if NumberOfBuildings is missing it will default to case 1 or 6 depending on how isAggregate is populated, and case 1 if they are both missing
